src/openloco/industrymgr.cpp

```diff
@@ -38,8 +38,6 @@ namespace openloco::industrymgr
                 }
             }
         }
-
-        call(0x00453234);
```

Comment:
This effectively meant industrymgr::update was being called twice in a tick, as you said. Does that mean industries were producing cargo twice as fast, too? This should probably be mentioned in the changelog.

Reply:
> Does that mean industries were producing cargo twice as fast, too?

I am not sure.

Reply:
Tested it a bit; doesn't appear to have that sort of effect. There's RNG involved, though, so it's a little hard to test.